src/components/descriptions/Descriptions.ts

```diff
@@ -5,11 +5,37 @@
  * Other licensing options may be available, please reach out to data-viz@feedzai.com for more information.
  */
 
-//when B or S are pressed the descriptions changed in the chart
-export function descriptionsChanger(ref, type, descs, title, autoDescriptions, event) {
-	title = title + ", " + type + ". " + (autoDescriptions ? "Automatic description: " : "");
+type AutoDescriptionsProps = {
+	dynamicDescriptions?: boolean;
+	apiKey: string;
+	model?: string;
+	temperature?: number;
+};
 
-	//When pressed reads the smaller description
+/**
+ * Handles the longer and shorter description change when Alt+B or Alt+S are pressed, respectively.
```

Comment:
The alt+B and alt+S command is not for MacOS. Can you add the "option" detail here and everywhere you mention shortcuts in the documentation?